recipes/rhessys/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015-2018, conda-forge
```

Comment:
You probably don't need to manually package the license file if it is in the source distribution you are using. Just point the entry in the `meta.yaml` to the location in the distribution directory.

Reply:
I pointed the License file to `https://github.com/DavidChoi76/rhessysec/blob/master/LICENSE.txt` but that occurred an error. So I want to keep the License file in this folder.